app/serializers/song_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class SongSerializer < ActiveModel::Serializer
+  attributes :id, :name, :spotify_uri
+
+  has_one :artist, embed: :id, :include => true
```

Comment:
And as @hchood mentioned in `ArtistSerializer` we'll similarly put the include in the json rendering.

Oddly, the documentation for ActiveModelSerializers doesn't document that way. Oh well.